instrumentation/opentelemetry-instrumentation-psycopg2/src/opentelemetry/instrumentation/psycopg2/__init__.py

```diff
@@ -152,6 +152,9 @@ def get_statement(self, cursor, args):
         statement = args[0]
         if isinstance(statement, Composed):
             statement = statement.as_string(cursor)
+        # Fix sql parameter placeholder(%s) to format sql parameters
+        if len(args) >= 2 and len(args[1]) > 0:
+            statement = statement % args[1]
```

Comment:
What if the statement does not include a "%"?

``` python
In [6]: "statement %s" % "test"
Out[6]: 'statement test'

In [7]: "statement " % "test"
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-29de3600cd4d> in <module>
----> 1 "statement " % "test"

TypeError: not all arguments converted during string formatting
```

Reply:
@caowenbin How does args look like? Does this `statement = statement % args[1]` cover all the cases?